### PR TITLE
3271 Apply flow June release - Type of Application page

### PR DESCRIPTION
**Pull request by shawn320**
### Description
Refactor Type of Application page for June release. Updated page content and added 2 new option

### Related Azure Boards Work Items
[AB#3271](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3271)

### Checklist

- [ ] I have tested the changes locally
- [ ] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [ ] I have checked that my code follows the project's coding style by running `npm run format:check`
- [ ] I have checked that my code contains no linting errors by running `npm run lint`
- [ ] I have checked that my code contains no type errors by running `npm run typecheck`
- [ ] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [ ] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx [hunk 1]
```diff
@@ -127,7 +130,24 @@ export default function ApplyFlowTypeOfApplication() {
...
+          <li>{t('apply:eligibility.type-of-application.senior')}</li>
+          <li>{t('apply:eligibility.type-of-application.valid-disability-tax-credit')}</li>
+          <li>{t('apply:eligibility.type-of-application.live-independently')}</li>
+        </ul>
+        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-child')}</h2>
+        <p>{t('apply:eligibility.type-of-application.apply-child-eligibility')}</p>
+        <ul className="mb-6 list-disc pl-6">
+          <li>{t('apply:eligibility.type-of-application.sixteen-or-older')}</li>
+          <li>{t('apply:eligibility.type-of-application.under-eighteen')}</li>
+        </ul>
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:146]: Add space between headings, paragrahps and lists and use same bullet list style like other pages.

```suggestion
        <div className="mb-6 space-y-4">
        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-self')}</h2>
        <p>{t('apply:eligibility.type-of-application.apply-self-eligibility')}</p>
        <ul className="list-disc space-y-1 pl-7">
          <li>{t('apply:eligibility.type-of-application.senior')}</li>
          <li>{t('apply:eligibility.type-of-application.valid-disability-tax-credit')}</li>
          <li>{t('apply:eligibility.type-of-application.live-independently')}</li>
        </ul>
        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-child')}</h2>
        <p>{t('apply:eligibility.type-of-application.apply-child-eligibility')}</p>
        <ul className="list-disc space-y-1 pl-7">
          <li>{t('apply:eligibility.type-of-application.sixteen-or-older')}</li>
          <li>{t('apply:eligibility.type-of-application.under-eighteen')}</li>
        </ul>
        </div>
```

File: frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx [hunk 2]
```diff
@@ -139,12 +159,22 @@ export default function ApplyFlowTypeOfApplication() {
             options={[
               {
                 value: ApplicantType.Personal,
-                children: t('apply:eligibility.type-of-application.radio-options.personal'),
+                children: <Trans ns={handle.i18nNamespaces} i18nKey="eligibility.type-of-application.radio-options.personal" />,
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:162]: Add namespace key prefix `apply:` to `i18nKey` prop. Do the same to others.
```suggestion
                children: <Trans ns={handle.i18nNamespaces} i18nKey="apply:eligibility.type-of-application.radio-options.personal" />,
```

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en/apply.json [hunk 3]
```diff
@@ -187,10 +187,25 @@
...
+      "apply-self-eligibility": "You can currently apply for yourself if you are:",
+      "senior": "65 or older",
+      "valid-disability-tax-credit": "18 to 64 and have a valid Disability Tax Credit certificate",
+      "live-independently": "16 and 17 and live independently from your parents",
+      "apply-child": "Applying on behalf of your child",
+      "apply-child-eligibility": "You can apply on behalf of your child if:",
+      "sixteen-or-older": "You are 16 or older and the parent or legal guardian of the child; and",
+      "under-eighteen": "The child is under 18 years old",
+      "form-instructions": "Who are you applying for?",
+      "note": "please note, only one application can be processed for each child.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/apply.json:201]: This paragraph is missing in the screen.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx [hunk 4]
```diff
@@ -127,7 +130,31 @@ export default function ApplyFlowTypeOfApplication() {
       </div>
       <div className="max-w-prose">
         {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
-        <p className="mb-6 italic" id="form-instructions">
+        <p className="mb-6">{t('apply:eligibility.type-of-application.page-description')}</p>
+        <div className="mb-6 space-y-4">
+          <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-self')}</h2>
+          <div>
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:136]: Remove this empty DIV

File: frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx [hunk 5]
```diff
@@ -127,7 +130,31 @@ export default function ApplyFlowTypeOfApplication() {
...
+            <p>{t('apply:eligibility.type-of-application.apply-self-eligibility')}</p>
+            <ul className="list-disc space-y-1 pl-7">
+              <li>{t('apply:eligibility.type-of-application.senior')}</li>
+              <li>{t('apply:eligibility.type-of-application.valid-disability-tax-credit')}</li>
+              <li>{t('apply:eligibility.type-of-application.live-independently')}</li>
+            </ul>
+          </div>
+          <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-child')}</h2>
+          <div>
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:145]: Remove this empty DIV

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx (hunk 4 above)
Comment (shawn320) [reply frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:136]: The vertical space apply between <p> and <ul> as well, is there an alternative way to remove the space?

Comment (sebastien-comeau) [reply frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx:136]: We want to be aligned with other pages style. It's ok.

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**